index.html

```diff
@@ -1,86 +1,162 @@
-        </div>
-        <div class="subscribe">
-            <h3 class="title">Subscribe Me</h3>
-            <div class="contain-form">
+            <div class="textNav">
+                <span class="underline">my stories</span>, as an <span class="underline">enginner</span>, <span class="underline">developer</span>, and <span class="underline">gamer</span>.
+            </div>
+            <div class="btnSearch">
+                <img src="image/search.png" alt="btnSearch">
+            </div>
+            <div class="searchContain">
                 <form action="">
-                    <input type="text" name="email" placeholder="Enter your email">
-                    <input type="submit" value="Subscribe">
+                    <input type="text" class="txtSearch" placeholder="search something...">
                 </form>
+            </div>                
+        </nav>
+        <div class="sliderPostContain container">
+            <div class="sliderPosts">
+                <div class="sliderPost">
+                    <div class="sliderPostImg">
+                        <img src="image/slider_1.png" alt="sliderPostImg">
+                    </div>
+                    <div class="sliderPostMain">
+                        <a href="" class="sliderPostTitle">Lorem ipsum dolor sit amet consectetur</a>
+                        <a href="" class="sliderPostCategory">Chuyện của tôi</a>
+                    </div>
+                </div>
+                <div class="sliderPost">
+                    <div class="sliderPostImg">
+                        <img src="image/slider_2.png" alt="sliderPostImg">
+                    </div>
+                    <div class="sliderPostMain">
+                        <a href="" class="sliderPostTitle">Lorem ipsum dolor sit amet consectetur</a>
+                        <a href="" class="sliderPostCategory">Chuyện của tôi</a>
+                    </div>
+                </div>
+                <div class="sliderPost">
+                    <div class="sliderPostImg">
+                        <img src="image/slider_3.png" alt="sliderPostImg">
+                    </div>
+                    <div class="sliderPostMain">
+                        <a href="" class="sliderPostTitle">Lorem ipsum dolor sit amet consectetur</a>
+                        <a href="" class="sliderPostCategory">Chuyện của tôi</a>
+                    </div>
+                </div>
+            </div>
+            <div class="swipe">
+                <span class="arrow arrowLeft">
+                    <img src="image/Arrow 1.png" alt="arrowSwipe">
+                </span>
+                <span class="swipeText">Swipe to discover more <span style="color: orange;">popular</span> stories</span>
+                <span class="arrow arrowRight">
+                    <img src="image/Arrow 1.png" alt="arrowSwipe">
+                </span>
             </div>
         </div>
-      </div>
-    </div>
-  </body>
+    </header>
+    
+    <main>
```

Comment:
I don't get the point of using \<main> and \<section> tags for? :((
Is it simpler if we use \<div> and assign it with an id?

Reply:
Again, whats \<main> tag for? :((